sentry/src/main/java/io/sentry/SentryOptions.java

```diff
@@ -1461,9 +1465,11 @@ public boolean isTraceSampling() {
    *
    * <p>Note: this is an experimental API and will be removed without notice.
    *
+   * @deprecated please use {{@link SentryOptions#addTracePropagationTarget(String)}} instead
    * @param traceSampling - if trace sampling should be enabled
    */
   @ApiStatus.Experimental
+  @Deprecated
```

Comment:
You can remove the experimental if its deprecated.

Reply:
Removed